domain/src/main/scala/BitcoinCryptography.scala

```diff
@@ -0,0 +1,13 @@
+package org.parsec.protocol
+
+trait BitcoinCryptography {
```

Comment:
Agreed but let us implement this after a couple of PRs I have in store which will clarify the domain further?